fixed misspellings (#2392)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/deprecation-of-site-mailboxes.md

  >[!NOTE]

>   > - Folder with no email items inside them will not be exported.

Reviewer: ```suggestion
  > - Folders with no email items inside them will not be exported.
```

>   > - The site mailboxes contain special folder named, "Documents", of type IPF.ShortcutFolder. This contains "links" to files that are on SP site. The actual SP files must be       exported using eDiscovery for SP sites.

Reviewer: ```suggestion
  > - The site mailboxes contain a special folder named, "Documents", of type IPF.ShortcutFolder. This contains "links" to files that are on SP sites. The actual SP files must be exported using eDiscovery for SP sites.
```

  > - Outlook shows the items inside the Documents folder as unsafe, this is an expected behavior.

>   > - The document attachments in the emails of Documents folder are just place holder, the actual documents are stored in SharePoint.

Reviewer: ```suggestion
  > - The document attachments in the emails of Documents folders are just placeholders, the actual documents are stored in SharePoint.
```

> [!NOTE]

> > - Folder with no email items inside them will not be exported.

Reviewer: ```suggestion
> - Folders with no email items inside them will not be exported.
```

> > - The site mailboxes contain special folder named, "Documents", of type IPF.ShortcutFolder. This contains "links" to files that are on SP site. The actual SP files must be       exported using eDiscovery for SP sites.

Reviewer: ```suggestion
> - The site mailboxes contain a special folder named, "Documents", of type IPF.ShortcutFolder. This contains "links" to files that are on SP sites. The actual SP files must be exported using eDiscovery for SP sites.
```

> - Outlook shows the items inside the Documents folder as unsafe, this is an expected behavior.

> > - The document attachments in the emails of Documents folder are just place holder, the actual documents are stored in SharePoint.

Reviewer: ```suggestion
> - The document attachments in the emails of Documents folders are just placeholders, the actual documents are stored in SharePoint.
```
